lce-rxjava3/build.gradle.kts

```diff
@@ -0,0 +1,25 @@
+plugins {
+    id("kotlin")
+    id("org.jetbrains.dokka")
+}
+
+apply {
+    from("$rootDir/.buildscript/configure-signing.gradle")
+}
+
+java {
+    toolchain {
+        languageVersion.set(JavaLanguageVersion.of(17))
+    }
+}
+kotlin {
+    jvmToolchain(17)
+}
+
+dependencies {
+    api(project(":lce"))
```

Comment:
side-note: would be nice to introduce `projects.lce` statically typed accessors.